pkg/controller/admissionchecks/multikueue/controllers.go

```diff
@@ -26,9 +26,11 @@ import (
 )
 
 const (
-	defaultGCInterval        = time.Minute
-	defaultOrigin            = "multikueue"
-	defaultWorkerLostTimeout = 5 * time.Minute
+	defaultGCInterval                 = time.Minute
+	defaultOrigin                     = "multikueue"
+	defaultWorkerLostTimeout          = 5 * time.Minute
+	defaultDispatcherName             = "kueue.x-k8s.io/multikueue-dispatcher-all-at-once"
+	incrementalDispatcherRoundTimeout = 5 * time.Minute
```

Comment:
Shouldn't we define this `incrementalDispatcherRoundTimeout` in workload.go?
Because this is used only in the file.